Panic when using arithmetic operations

**To Reproduce**
```
package main

func main()() {
    var f f32 = 1.0 + 1.0 * 1.0
    test(f, 2.0, "f != 2.0")
}
```

**Expected behavior**
Test pass.

**Screenshots**
```
panic: runtime error: index out of range

goroutine 1 [running, locked to thread]:
github.com/SkycoinProject/cx/cxgo/actions.ShorthandExpression(0xc0000a20f8, 0x1, 0x1, 0xc0000a2178, 0x1, 0x1, 0x9, 0xc0000a2178, 0x1, 0x1)
    /home/user/go/src/github.com/SkycoinProject/cx/cxgo/actions/expressions.go:296 +0x629
github.com/SkycoinProject/cx/cxgo/parser.yyParse(0x1019bc0, 0xc0001b0100, 0x0)
    /home/user/go/src/github.com/SkycoinProject/cx/cxgo/parser/cxgo.go:2183 +0x6ea0
github.com/SkycoinProject/cx/cxgo/parser.Parse(...)
    /home/user/go/src/github.com/SkycoinProject/cx/cxgo/parser/cxgo.go:18
main.ParseSourceCode(0xc000010028, 0x1, 0x1, 0xc0000322b0, 0x1, 0x1)
    /home/user/go/src/github.com/SkycoinProject/cx/cxgo/main.go:768 +0x3f5
main.parseProgram(0x0, 0xe81578, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    /home/user/go/src/github.com/SkycoinProject/cx/cxgo/main.go:819 +0x3a7
main.main()
    /home/user/go/src/github.com/SkycoinProject/cx/cxgo/main.go:1089 +0x396
```

**Desktop (please complete the following information):**
 - OS: linux
 - CX Version : 0.7.1 commit : https://github.com/SkycoinProject/cx/commit/1e849f9cba7698941615e6320fac45bf741b8af2

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -24,6 +24,8 @@
   * #419: Initializing array or slice with values equivalent to nil.
   * #481: Array literals: Problem with temporary variables in multi-dimensional arrays.
   * #482: Slice literals: Problem with temporary variables in multi-dimensional slices.
+  * #131: Panic when using arithmetic operations.
+
 * Documentation
   * ...
 * Miscellaneous

--- a/cxgo/actions/expressions.go
+++ b/cxgo/actions/expressions.go
@@ -285,7 +285,8 @@ func ShorthandExpression(leftExprs []*CXExpression, rightExprs []*CXExpression, 
 	case OP_ADD:
 		// Handling special case of arguments being strings.
 		// In this case we use `str.concat`.
-		if rightExprs[len(rightExprs)-1].Outputs[0].Type == TYPE_STR {
+		rightLen := len(rightExprs) - 1
+		if rightLen >= 0 && len(rightExprs[rightLen].Outputs) > 0 && rightExprs[rightLen].Outputs[0].Type == TYPE_STR {
 			operator = Natives[OP_STR_CONCAT]
 		} else {
 			operator = Natives[OP_UND_ADD]

